MCprep_addon/tracking.py

```diff
@@ -484,18 +485,19 @@ def string_trunc(self, value):
 
 class TRACK_OT_toggle_enable_tracking(bpy.types.Operator):
 	"""Enabled or disable usage tracking"""
-	bl_idname = IDNAME+".toggle_enable_tracking"
+	bl_idname = f"{IDNAME}.toggle_enable_tracking"
 	bl_label = "Toggle opt-in for analytics tracking"
-	bl_description = "Toggle anonymous usage tracking to help the developers. "+\
-			" The only data tracked is what MCprep functions are used, key "+\
-			"blender/addon information, and the timestamp of the addon installation"
+	bl_description = (
+		"Toggle anonymous usage tracking to help the developers. "
+		"The only data tracked is what MCprep functions are used, key blender"
+		"/addon information, and the timestamp of the addon installation")
 	options = {'REGISTER', 'UNDO'}
 
-	tracking = bpy.props.EnumProperty(
-		items = [('toggle', 'Toggle', 'Toggle operator use tracking'),
+	tracking: bpy.props.EnumProperty(
+		items=[('toggle', 'Toggle', 'Toggle operator use tracking'),
 				('enable', 'Enable', 'Enable operator use tracking'),
 				('disable', 'Disable', 'Disable operator use tracking (if already on)')],
```

Comment:
This I'm curious about, since it says `opt-in` analytics. However, from my understanding of `bpy`, Blender will by default use the first item in an enum. If that's the case, we should make sure that it's truly opt in